source/includes/steps-fle-convert-to-a-remote-master-key-aws.yaml

```diff
@@ -10,10 +10,22 @@ content: |
      - **access key ID**
      - **secret access key**
 
-  2. Grant the IAM user narrowly scoped ``List`` and ``Read`` permissions only for the KMS
-     service. See Amazon's official documentation on
-     `Adding permissions to a user <https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_change-permissions.html>`__
-     to set these permissions.
+  2. Grant your IAM user ``kms:Encrypt`` and ``kms:Decrypt`` permissions for
+     your remote master key. The following inline 
+     policy allows an IAM user to perform CSFLE with the least privileges
+     possible:
+
+     .. literalinclude:: /includes/iam-user-policy-minimum.json
+        :language: json
+
+     .. note:: Remote Master Key Amazon Resource Name (ARN)
+
+        The preceding policy requires the ARN of the key you generate in the
+        :ref:`Create the Master Key <aws-create-master-key>` step of this guide.
```

Comment:
Issue:
I noticed the step to which this links mentions the following in sub-step 2:
"In the Usage Permissions step of the key generation process, add the full KMS List and Read permissions to the IAM user you created in the previous step. This authorizes the user to encrypt and decrypt the new master key."
This conflicts with the text you've added in this step.

Suggestion: There doesn't need to be any skipping around for the tutorial. You can apply a policy to an IAM user after you create it, so the example policy and info about user permissions can be moved here. At least remove the conflicting/duplicate information from the create master key sub-step and either address or ticket the rest, which does seem in-scope for this ticket.

Reply:
The policy created in the step to which this links is a [Key Policy](https://docs.aws.amazon.com/kms/latest/developerguide/key-policies.html), where as the step created for the user is an [IAM policy.](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_create.html)

AFAICT, you can configure a Key Policy to defer to an IAM policy for granting access, but you MUST grant an IAM user access to KMS operations, regardless of what is contained in the Key policy.

What is currently on the page seems to incorrectly discuss the two policies as equivalent (and doesn't mention that the default AWS KMS policy defers to IAM policies). I've updated the text to more clearly distinguish between IAM and KMS policies.

Reply:
I see, that makes sense. It wasn't clear to me in the description, and I think the policy listed there was for the IAM user in the revision I commented on. Let me know if you want another copy review if you incorporate any changes based on the info you receive from the external review.